<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>860</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Heavy Ball</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/img/boule.svg</normaloff>:/img/boule.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <item row="12" column="1">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignTop">
     <widget class="QPushButton" name="pesee">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Peser !</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3" alignment="Qt::AlignHCenter">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>662</width>
        <height>158</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>622</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>662</width>
         <height>158</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0" alignment="Qt::AlignHCenter">
         <widget class="Conteneur" name="conteneur">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>662</width>
            <height>94</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>662</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="7" column="0">
     <layout class="QGridLayout" name="layout_gauche">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="1" alignment="Qt::AlignBottom">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>28</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/balance_tige.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="pied_gauche_bas">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="viderGauche">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="statusTip">
         <string>Remet toutes les boules du plateau gauche dans le conteneur.</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="pied_gauche_haut">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>197</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/balance_plateau_gauche.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" alignment="Qt::AlignRight|Qt::AlignBottom">
       <widget class="Plateau" name="plateau_1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>197</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>197</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="2">
     <layout class="QGridLayout" name="layout_droite">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="2" column="0" alignment="Qt::AlignBottom">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>28</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>28</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/balance_tige.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="viderDroit">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="statusTip">
         <string>Remet toutes les boules du plateau droite dans le conteneur.</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>197</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/img/balance_plateau_droite.svg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignBottom">
       <widget class="Plateau" name="plateau_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>197</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>197</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer name="pied_droite_bas">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <spacer name="pied_droite_haut">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="10" column="0" colspan="3" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="donner_reponse">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Donner la réponse</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuJeu">
    <property name="title">
     <string>Jeu</string>
    </property>
    <addaction name="actionNouvelle_Partie"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="action_propos"/>
   </widget>
   <widget class="QMenu" name="menuParam_tres">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuMasse_de_la_boule_d_fectueuse">
     <property name="title">
      <string>Masse de la boule défectueuse</string>
     </property>
     <addaction name="actionAl_atoire"/>
     <addaction name="actionPlus_l_g_re"/>
     <addaction name="actionPlus_lourde"/>
     <addaction name="separator"/>
     <addaction name="actionN_est_pas_multiple"/>
     <addaction name="actionPeut_etre_un_multiple"/>
     <addaction name="actionEst_un_multiple"/>
     <addaction name="actionD_finir_coefficient"/>
    </widget>
    <addaction name="actionNombre_de_boules"/>
    <addaction name="actionNombre_de_pes_e"/>
    <addaction name="actionRemiseAZero"/>
    <addaction name="menuMasse_de_la_boule_d_fectueuse"/>
   </widget>
   <addaction name="menuJeu"/>
   <addaction name="menuParam_tres"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_propos">
   <property name="text">
    <string>À propos ...</string>
   </property>
   <property name="statusTip">
    <string>Donne des informations sur le programme, les crédits et la licence.</string>
   </property>
  </action>
  <action name="actionNouvelle_Partie">
   <property name="text">
    <string>Nouvelle Partie</string>
   </property>
   <property name="statusTip">
    <string>Démarre une nouvelle partie.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="statusTip">
    <string>Ferme le jeu.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRemiseAZero">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remise à zéro de la balance</string>
   </property>
   <property name="statusTip">
    <string>Définit si la balance se remet en position initiale lorsque l'on modifie les plateaux.</string>
   </property>
  </action>
  <action name="actionNombre_de_pes_e">
   <property name="text">
    <string>Nombre de pesées ...</string>
   </property>
   <property name="statusTip">
    <string>Définit le nombre de pesées. (Par défaut : 3)</string>
   </property>
  </action>
  <action name="actionNombre_de_boules">
   <property name="text">
    <string>Nombre de boules ...</string>
   </property>
   <property name="statusTip">
    <string>Définit le nombre de boules. (Par défaut : 12)</string>
   </property>
  </action>
  <action name="actionN_est_pas_multiple">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>N'est pas multiple</string>
   </property>
   <property name="statusTip">
    <string>Si la boule est plus lourde, sa masse NE PEUT PAS ÊTRE égale à la somme de plusieurs autres.</string>
   </property>
  </action>
  <action name="actionPeut_etre_un_multiple">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Peut être un multiple</string>
   </property>
   <property name="statusTip">
    <string>Si la boule est plus lourde, sa masse PEUT ÊTRE égale à la somme de plusieurs autres.</string>
   </property>
  </action>
  <action name="actionEst_un_multiple">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Est un multiple</string>
   </property>
   <property name="statusTip">
    <string>Si la boule est plus lourde, sa masse EST égale à la somme de plusieurs autres.</string>
   </property>
  </action>
  <action name="actionD_finir_coefficient">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Définir coefficient ...</string>
   </property>
   <property name="statusTip">
    <string>Définit le coefficient de la boule plus lourde que les autres. (Par défaut : aléatoire entre [2;12[ )</string>
   </property>
  </action>
  <action name="actionAl_atoire">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Aléatoire</string>
   </property>
  </action>
  <action name="actionPlus_l_g_re">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plus légère</string>
   </property>
  </action>
  <action name="actionPlus_lourde">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Plus lourde</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plateau</class>
   <extends>QGroupBox</extends>
   <header>plateau.h</header>
   <container>1</container>
   <slots>
    <signal>conteneurChanged()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>Conteneur</class>
   <extends>QGroupBox</extends>
   <header>conteneur.h</header>
   <container>1</container>
   <slots>
    <signal>conteneurChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pesee</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>peser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>426</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNouvelle_Partie</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>menuNiveau()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_propos</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>aPropos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plateau_1</sender>
   <signal>conteneurChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetBalance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viderGauche</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>viderGauche()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>449</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viderDroit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>viderDroit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemiseAZero</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setResetBalance(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNombre_de_pes_e</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>askNombrePesees()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNombre_de_boules</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>askNombreBoules()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plateau_2</sender>
   <signal>conteneurChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetBalance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionN_est_pas_multiple</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setNotMultiple(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPeut_etre_un_multiple</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setCanBeMultiple(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEst_un_multiple</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setMustBeMultiple(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionD_finir_coefficient</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>askCoefficient()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plateau_1</sender>
   <signal>conteneurChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>ajusterPlateaux()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plateau_2</sender>
   <signal>conteneurChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>ajusterPlateaux()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>646</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>donner_reponse</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>askReponse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>529</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAl_atoire</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setAleatoire(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlus_l_g_re</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setPlusLegere(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPlus_lourde</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setPlusLourde(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>peser()</slot>
  <slot>resetBalance()</slot>
  <slot>menuNiveau()</slot>
  <slot>a_propos()</slot>
  <slot>viderGauche()</slot>
  <slot>viderDroit()</slot>
  <slot>setResetBalance(bool)</slot>
  <slot>askNombrePesees()</slot>
  <slot>askNombreBoules()</slot>
  <slot>setNotMultiple(bool)</slot>
  <slot>setCanBeMultiple(bool)</slot>
  <slot>setMustBeMultiple(bool)</slot>
  <slot>askCoefficient()</slot>
  <slot>ajusterPlateaux()</slot>
  <slot>askReponse()</slot>
  <slot>setAleatoire(bool)</slot>
  <slot>setPlusLegere(bool)</slot>
  <slot>setPlusLourde(bool)</slot>
 </slots>
</ui>
